sdk/src/transaction/versioned/sanitized.rs

```diff
@@ -8,9 +8,9 @@ use {
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct SanitizedVersionedTransaction {
```

Comment:
`pub`ing these fields is my remaining concern.
I don't think we should do this, as it makes it possible to:

1. modify these fields in external code
2. construct a **sanitized** transaction direction

2 is more my concern. Right now with these fields as private, which means the only way we can create a `SanitizedVersionedTransaction` is by going through the sanitization checks, which imo we want to keep.

Not sure about naming, but perhaps we can do something along the lines of adding a method to `SanitizedVersionedTransaction`:

```rust
/// Consumes the SanitizedVersionedTransaction, returning the fields individually.
pub fn destructure(self) -> (Vec<Signature>, SanitizedVersionedMessage) {
    (self.signatures, self.message)
}

which we then use inside `RuntimeTransactionStatic::try_from`.

Reply:
in 43eda3916141f8f4541ceca02a40d3146ee75967